File: raw_python/var.py
"""Variable types"""
# %%
# ---------------- Simple ----------------
a: int = 99
a
# %%
b: float = 99.99
b
# %%
c: bool = True
c
# %%
d: str = 'h'
d
# %%
a: str = "hi"
a
# %%
e: complex = 1 + 2j
e
# %%
# ---------------- Composite ----------------
f = (9)
type(f)
# %%
f = (9,)
type(f)
# %%
g = [1, '1', 1.0, True]
g
# %%
a = {"hi": 99, "pussy": 88, "adiyogi": 128}
print(a.get("hii", '?'))
# %%
b = {"bhairavi": 9 == 11}
b.get("bhairavi", '?')

File: JS/var.js
// Variable types

// ---------------- Simple ----------------
// JavaScript doesn't have type annotations like Python
// but we can use similar variables
let a = 99;
console.log(a); // 99

let b = 99.99;
console.log(b); // 99.99

let c = true;
console.log(c); // true

let d = 'h';
console.log(d); // 'h'

// In JavaScript, variables can be reassigned to different types
a = "hi";
console.log(a); // "hi"

// ---------------- Composite ----------------
let myNameArray = ["Chris", "Bob", "Jim", c];
console.log(myNameArray); // ["Chris", "Bob", "Jim"]

let dog = { name: "Spot", breed: "Dalmatian" };
console.log(dog); // { name: "Spot", breed: "Dalmatian" }